docs/Reserved-Aliases.md

```diff
@@ -1,6 +1,107 @@
-The following are [reserved aliases](#reserved-aliases)  in template engine. They are applied to .NET SDK 7+.
+### Reference for dotnetcli.host.json
+Generally the name of parameter symbol is the long name and its first character is the short name. Prefixing long name with `--` as long alias and prefixing short name with `-` are the aliases for the parameter symbol listed in the template options.  
+With `dotnetcli.host.json` file we can change long alias and short alias of the template option and its visibility. `dotnetcli.host.json` is located at the same directory as template.json file. It has the following configuration.
+- symbolsInfo  
+It has a collection of the name of specified parameter symbol in template.json with corresponding long or short name to override original ones and the visibility of the template option.  
+   - longName
+   - shortName  
+   Empty means short alias should not be used.
+   - isHidden  
+   True if hiding the parameter in CLI.
+- usageExamples  
+It's an array of usage exmples. Currently it's not used and please track on issue [#3262](https://github.com/dotnet/templating/issues/3262).
 
-If reserved alias is used as the name of Parameter symbol, it will be prefixed with `param:` or `-p:` as template option when instantiating the template.
+The following examples are several cases when using `dotnetcli.host.json`.
+
+##### Examples
+Override original long alias `--TargetFrameworkOverride` with `--targetframework` and override short alias `-T` with `-tf`. And this template option is hidden in CLI help information.
```

Comment:
```suggestion
Overrides original long alias `--TargetFrameworkOverride` with `--targetframework` and overrides short alias `-T` with `-tf`.
```

I suggest to remove `isHidden` to separate example. It doesn't make sense to override the aliases, and then hide the option.